src/FastPriorityQueue.php

```diff
@@ -41,63 +40,61 @@ class FastPriorityQueue implements Iterator, Countable, Serializable
     public const EXTR_BOTH     = PhpSplPriorityQueue::EXTR_BOTH;
 
     /** @var self::EXTR_* */
-    protected $extractFlag = self::EXTR_DATA;
+    protected int $extractFlag = self::EXTR_DATA;
```

Comment:
We should take the occasion to make many of these `final` BTW - the pain of dealing with `protected` is quite high.

That said, feel free to dismiss if out of scope here.